package/kedro_viz/data_access/managers.py

```diff
@@ -47,10 +47,15 @@ def __init__(self):
         self.nodes = GraphNodesRepository()
         self.registered_pipelines = RegisteredPipelinesRepository()
         self.tags = TagsRepository()
-        self.modular_pipelines = ModularPipelinesRepository()
-        # Make sure each registered pipeline has a distinct collection of edges.
+
+        # Make sure each registered pipeline has a distinct collection of
```

Comment:
As you can see, previously, each registered pipeline keeps its own collection of nodes and edges. Using the same pattern for the modular pipelines is therefore natural.

Reply:
thanks, this def feels more natural!